libnavigation-core/src/test/java/com/mapbox/navigation/core/routealternatives/RouteAlternativesControllerTest.kt

```diff
@@ -403,17 +403,17 @@ class RouteAlternativesControllerTest {
             )
         )
 
-        val routeProgressSlot = slot<RouteProgress>()
-        val alternativesSlot = slot<List<DirectionsRoute>>()
-        val routerOriginSlot = slot<RouterOrigin>()
+        val routeProgressSlots = mutableListOf<RouteProgress>()
```

Comment:
why need to replace slot by list if need the latest item only? 🤔

Reply:
It is a new limitation in mockk library: https://github.com/mockk/mockk/releases/tag/1.10.3:

> issue [#352](https://github.com/mockk/mockk/issues/352) prevent slots from being used when verifying multiple calls